app/controllers/application_controller.rb

```diff
@@ -6,6 +6,7 @@ class ApplicationController < ActionController::Base
 
   before_action :load_site_configuration
   before_action :logout_inactive
+  before_action :terms_of_use
```

Comment:
Does this need to run before every action in this controller?

Reply:
My bad. I try to do it like in self registration. But forgot to look on other variables in application.html.haml. So i edit it and use existing @site there and remove before_action and terms_of_use function from application controller.